README.md

```diff
@@ -32,3 +32,15 @@ $ architect dev architect.yml
 ```
 
 Once the deploy has completed, you can reach your new service by going to https://web.localhost.architect.sh/.
+
+## Deploying to the Cloud
+
+Want to try deploying this to a cloud environment? Architect's got you covered there, too! It only takes a minute to
+[sign up for a free account](https://cloud.architect.io/signup).
+
+You can then [deploy the application](https://docs.architect.io/getting-started/introduction/#deploy-to-the-cloud). To do this, run the following command. "example-environment" is the name of the free environment that is created when you register with Architect.
```

Comment:
Change to (keep the hyper link) -
You can then deploy the application by running the command below. Note that “example-environment” is the free environment that is created with your Architect account.